src/Scripting/Core/Hosting/ObjectFormatter/CommonPrimitiveFormatter.Options.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Scripting.Hosting
+{
+    public abstract partial class CommonPrimitiveFormatter
+    {
+        public struct Options
+        {
+            public readonly bool UseHexadecimalNumbers;
```

Comment:
Per our offline conversation, this comment is suggesting that the `NumberRadix` type should be used in place of `bool` in all public APIs.